### ISSUE TITLE
[BUG] Steps form - odd navigation when "enter" pressed on input

### ISSUE THREAD

**Issue by Christopher-O** [wet-boew/wet-boew#9106]
## Required information

**Describe the bug**
When you press enter on an input on any fieldset input after the second fieldset...the second fieldset opens up as well as the fieldset area you are on also stays open.  Our group is working on an example that has 6 fieldsets and any input in fieldset 3 to 6 will produce the same issue (second fieldset opens)

**To Reproduce**
1. Go to 'https://wet-boew.github.io/v4.0-ci/demos/stepsform/stepsform-en.html'
2. Fill out the required fields until step 3
3. Select one of the radio buttons
4. Press the enter key
5. See error (The second fieldset is focused on)

**Expected behavior**
Either the fieldset area will validate and draw an error if a required field is not filed in correctly, or should do nothing.

I am asking for Principal publisher to please do the work

**Version of WET-BOEW/GCWeb you are using**
e.g. v4.0.42.2 (this error is visible from WET4 style guide)

**Desktop/Smartphone (please complete the following information)**
chrome (90.0.4430.212), firefox (78.10)

**Screenshots**
![image](https://user-images.githubusercontent.com/9367162/118712845-806a3980-b7de-11eb-961a-70089407a623.png)

**Comment by thekodester** [comment]
Happens on any input, pressing enter on preferred phone number caused it to go back to step 1.

**Comment by crochefort** [comment]
@duboisp je confirme le bug dans mon projet aussi!
Sa serait bien que PP s'occupe du bug!

**Comment by thekodester** [comment]
I don't think that is the issue, running some test, pressing enter is triggering the previous button.

**Comment by thekodester** [comment]
Tried playing around with [keypress](https://api.jquery.com/keypress/) call inside the setStepsBtnEvent:
https://github.com/wet-boew/wet-boew/blob/1ad91b1bdf9ec5814c580b0883c91b0ea6f69e13/src/plugins/stepsform/stepsform.js#L155

But seemed to cause more bugs, I think switching the order of the buttons would also work, but unsure how that would affect accessibility.

**Comment by andrewdp** [comment]
I think this should be fixed with #9459
